plugins/auth-backend/src/providers/cloudflare-access/provider.ts

```diff
@@ -153,6 +153,7 @@ export type CloudflareAccessResult = {
   claims: CloudflareAccessClaims;
   cfIdentity: CloudflareAccessIdentityProfile;
   expiresInSeconds?: number;
+  jwt: string;
```

Comment:
I'm thinking I'd prefer `token` here, or `accessToken`/`idToken` if that is what it is. JWT is a bit of an implementation detail.